src/astProcessor.ts

```diff
@@ -1,5 +1,5 @@
-//const Connection = require('database-js').Connection;
 import { Connection } from 'database-js'
+import { Element, Feature } from './interfaces'
 
 export class AstProcessor {
   private async getFileContent(filePath) : Promise<any> {
```

Comment:
- [ ]  since the AST file content is being filtered, it may be better renaming it to `filterAstFile`
- [ ]  declare `filePath` type (string)
- [ ]  put the `new` statement inside the `try`
- [ ]  rethrow the error (never log it) - remember to close the connection before doing it